SRS/SRS_GRIDDED/backlog_pipeline_reprocessing.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to copy and reprocess n_files_limit(default is 20) GHRSST files
+via pipeline in order not to overflow the celery queue with backlog items to
+reprocess. New files being uploaded by the facility will still be processed
+within a reasonable time frame of couple of minutes max
+
+This script should be run in a tmux/screen/background as it will take a long
+long time to run
+roughly 770000 files (10 sec/file) = 3 months of reprocessing
```

Comment:
this actually writes a log files of files already pushed to $WIP_DIR. This log is read at each iteration, ie every 20 files